com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDEditorUtils.cs

```diff
@@ -234,7 +234,7 @@ internal static void DrawDecalLayerMask_Internal(Rect rect, GUIContent label, Se
             EditorGUI.BeginProperty(rect, label, property);
 
             EditorGUI.BeginChangeCheck();
-            int changedValue = EditorGUI.MaskField(rect, label ?? GUIContent.none, property.intValue, HDRenderPipeline.defaultAsset.decalLayerNames);
+            int changedValue = EditorGUI.MaskField(rect, label ?? GUIContent.none, property.intValue, HDRenderPipeline.currentAsset.decalLayerNames);
```

Comment:
Shouldn't this be `HDDefaultSettings.instance`? or can we have multiple decal layer name per HDRP assets?

(same thing for the light layer below)

Reply:
yes! good point :) will update

Reply:
Updated in last commit.